### PR TITLE
Apply metadata changes for NSIDC ingest and others

### LINKED ISSUE #71: Ensure satellite img dictionarys have consistent metadata
sometimes they come as string and sometimes as numeric values, so @mliukis has them all listed when grouping in composites:
```
LANDSAT89 = MSTuple('L89', ['8.', '9.', '8.0', '9.0', 8.0, 9.0], 'L8_L9')
SENTINEL1 = MSTuple(‘S1’, ['1A', '1B’], ‘S1A_S1B’)
SENTINEL2 = MSTuple(‘S2’, ['2A', '2B'], ‘S2A_S2B’)
```

### PR DESCRIPTION
**Pull request by jhkennedy**
Fixes #71 

Discussed here: https://itslive.slack.com/archives/CA1U7JACS/p1669060017258639?thread_ts=1669051774.090769&cid=CA1U7JACS

Also, referenced in nasa-jpl/its_live_production#15

### CODE REVIEW

**Review by mliukis — CHANGES_REQUESTED** [review]
@jhkennedy I do see the corrections to the `mapping` data variable, so please ignore my previous comment :)

File: netcdf_output.py [hunk 1]
```diff
@@ -1191,8 +1199,9 @@ def netCDF_packaging(VX, VY, DX, DY, INTERPMASK, CHIPSIZEX, CHIPSIZEY, SSM, SSM1
     var = nc_outfile.createVariable('interp_mask', np.dtype('uint8'), ('y', 'x'), fill_value=0,
                                     zlib=True, complevel=2, shuffle=True, chunksizes=ChunkSize)
     var.setncattr('standard_name', 'interpolated_value_mask')
-    var.setncattr('description', 'light interpolation mask')
-    var.setncattr('units', 'binary')
+    var.setncattr('description', 'true where values have been interpolated')
+    var.setncattr('flag_values', [np.uint8(0), np.uint8(1)])
+    var.setncattr('flag_meanings', 'measured, interpolated')
```
[the remark this answers is not in the document]
Comment (jhkennedy) [reply netcdf_output.py:1204]: :eyes: wat. So even in their examples, values are comma-separated, but descriptions a space-separated :facepalm: 
https://cfconventions.org/Data/cf-conventions/cf-conventions-1.8/cf-conventions.html#flags

It should still be interpreted correctly as the conventions do say:
> If the string contains any commas, it is assumed to be a comma-separated list.

https://cfconventions.org/Data/cf-conventions/cf-conventions-1.8/cf-conventions.html#identification-of-conventions

:roll_eyes:

[the remark this answers is not in the document]
Comment (jhkennedy) [reply netcdf_output.py:1204]: I'll fix that here and on our end, but the Greenland Landsat run will have commas here, so will need to be fixed.

[the remark this answers is not in the document]
Comment (jhkennedy) [reply netcdf_output.py]: @mliukis I just pushed a change to space separated. It looks like this is the only place in the script that needs updating.

**Review by mliukis — APPROVED** [review]

**Pull request merged**